Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3408.cs

```diff
@@ -0,0 +1,212 @@
+			};
+			var btn2 = new Button
+			{
+				Text = "Change Property",
+				AutomationId = "btn2",
+				Command = new Command(() =>
+				{
+
+					foreach (var item in GetRecommendations(aacountListView.BindingContext))
+					{
+						item.Name = "New Item Name";
+						item.IsBusy = !item.IsBusy;
+					}
+
+				})
+			};
+			grd.Children.Add(aacountListView);
+			Grid.SetRow(aacountListView, 0);
+			grd.Children.Add(btn);
+			Grid.SetRow(btn, 1);
+			grd.Children.Add(btn2);
+			Grid.SetRow(btn2, 2);
+			Content = grd;
+		}
+
+#if UITEST
+		[Test]
+		public void Issue3408Test ()
+		{
+			RunningApp.WaitForElement (q => q.Marked ("btn1"));
+			RunningApp.WaitForElement (q => q.Marked ("Click to Change"));
+			RunningApp.Tap(q => q.Marked("btn1"));
+			RunningApp.WaitForElement(q => q.Marked("This should have changed"));
+			RunningApp.Tap(q => q.Marked("btn2"));
+			RunningApp.WaitForElement(q => q.Marked("New Item Name"));
+		}
+#endif
+
+		[Preserve(AllMembers = true)]
+		public class RecommendationsBaseViewModel : ViewModelBase
+		{
+			public string AccountName => $"";
+			public List<Recommendation> Recommendations { get; set; }
+		}
+
+		[Preserve(AllMembers = true)]
+		public class RecommendationsViewModel : RecommendationsBaseViewModel
+		{
+			public string AccountName => $"Recommendations";
+
+			public RecommendationsViewModel()
+			{
+				Recommendations = new List<Recommendation>()
+			{
+					new Recommendation(){ Name = "Click to Change"} ,
+					new Recommendation(){ Name = "Recommendations"} ,
+					new Recommendation(){ Name = "Recommendations"} ,
+			};
+			}
+		}
+
+		[Preserve(AllMembers = true)]
+		public class RecommendationsViewModel2 : RecommendationsBaseViewModel
+		{
+			public string AccountName => $"Recommendations 2";
+			public RecommendationsViewModel2()
+			{
+				Recommendations = new List<Recommendation>()
+			{
+					new Recommendation(){ Name = "This should have changed"} ,
+					new Recommendation(){ Name = "Recommendations 2"} ,
+					new Recommendation(){ Name = "Recommendations 2", IsBusy = true } ,
+			};
+			}
+		}
+
+		[Preserve(AllMembers = true)]
+		public class Recommendation : ViewModelBase
+		{
+			string _name;
+			public string Name
+			{
+				get { return _name; }
+				set
+				{
+					if (_name == value)
+						return;
+					_name = value;
+					OnPropertyChanged();
+				}
+			}
+		}
+
+	}
+
+	[Preserve(AllMembers = true)]
+	public class RecommendationsView : ContentView
+	{
+		public RecommendationsView()
+		{
+			Grid grd = new Grid();
+			var lst = new ListView
+			{
+				ItemTemplate = new DataTemplate(() =>
+				{
+					var swittch = new SwitchCell();
+					swittch.SetBinding(SwitchCell.TextProperty, new Binding("Name"));
+					swittch.SetBinding(SwitchCell.OnProperty, new Binding("IsBusy"));
+					return swittch;
+				})
+
+			};
+
+			lst.SetBinding(ListView.ItemsSourceProperty, new Binding("Recommendations"));
+			grd.Children.Add(lst);
+			Content = grd;
+		}
+
+		object context = null;
```

Comment:
Add a reference to this issue
https://github.com/xamarin/Xamarin.Forms/issues/3602

So future people know why this is here